### PR TITLE
stabilization for sprint 27

### PR DESCRIPTION
**Pull request by anup-nehe**
stabilization for sprint 27

### CODE REVIEW

**Review by kamalsinghthoughtworks — APPROVED** [review]

File: injitest/androidRegression.xml [hunk 1]
```diff
@@ -96,6 +96,7 @@
                 <methods>
                     <include name="noCardsAvailableToShare"/>
                     <include name="verifyReceivedCardTabPresent"/>
+                    <include name="verifyPopupsForShareWithSelfi"/>
```
Comment (swatigoel) [inline injitest/androidRegression.xml:99]: spelling typo mistake verifyPopupsForShareWithSelfi should be `verifyPopupsForShareWithSelfie`

File: injitest/src/main/java/inji/pages/AddNewCardPage.java [hunk 2]
```diff
@@ -19,8 +19,8 @@ public class AddNewCardPage extends BasePage{
     @iOSXCUITFindBy(accessibility = "goBack")
     private WebElement backButton;
     
-    @AndroidFindBy(accessibility = "issuer-ESignet")
-    @iOSXCUITFindBy(accessibility = "issuer-ESignet")
+    @AndroidFindBy(accessibility = "issuerHeading-ESignet")
+    @iOSXCUITFindBy(accessibility = "issuerHeading-ESignet")
     private WebElement downloadViaEsignet;
```
Comment (swatigoel) [inline injitest/src/main/java/inji/pages/AddNewCardPage.java:24]: This should be `issuerHeading-Mosip` right?

File: injitest/src/main/java/inji/pages/DetailedVcViewPage.java [hunk 3]
```diff
@@ -203,7 +207,16 @@ public boolean isEsignetLogoDisplayed() {
...
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+        clickOnElement(moreOptionsButton);
+    }
+    public boolean reVerificationInProgressText(String name) {
+        By fullName = By.xpath("//*[contains(@text,'" + name + "is verified successfully and now available for activation.')]");
+        return isElementDisplayed(fullName,50);
```
Comment (swatigoel) [inline injitest/src/main/java/inji/pages/DetailedVcViewPage.java:219]: are we able to automate this scenario?

File: injitest/src/main/java/inji/pages/AddNewCardPage.java (hunk 2 above)
Comment (anup-nehe) [reply injitest/src/main/java/inji/pages/AddNewCardPage.java:24]: updated

File: injitest/src/main/java/inji/pages/DetailedVcViewPage.java (hunk 3 above)
Comment (anup-nehe) [reply injitest/src/main/java/inji/pages/DetailedVcViewPage.java]: not automating this scenario

**Review by swatigoel — APPROVED** [review]

**Pull request merged**